### PR TITLE
update tiddit/sv

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -86,6 +87,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#567](https://github.com/nf-core/sarek/pull/567) - Fix interval name resolving during scatter/gather by moving logic to modules.config causing name to be correctly resolved on process execution; also fixed duplicate naming when variant callers produce multiple vcf files by adding field `type` to `meta` map
 - [#585](https://github.com/nf-core/sarek/pull/585) - Fix Spark usage for GATK4 modules
 - [#587](https://github.com/nf-core/sarek/pull/587) - Fix issue with VEP extra files
+- [#581](https://github.com/nf-core/sarek/pull/581) - `TIDDIT` is back
```
Comment (FriederikeHanssen) [inline CHANGELOG.md:90]: 🚀 🎉

**Review by FriederikeHanssen — APPROVED** [review]

**Comment by FriederikeHanssen** [comment]
oh: Tiddit test still needs to be include in the `ci.yml` to run

File: workflows/sarek.nf [hunk 2]
```diff
@@ -759,6 +759,7 @@ workflow SAREK {
         GERMLINE_VARIANT_CALLING(
             params.tools,
             cram_variant_calling_status_normal,
+            [],
```
[the remark this answers is not in the document]
Comment (maxulysse) [reply workflows/sarek.nf:762]: for now, just need to figure out how to use the channel if available

File: subworkflows/nf-core/variantcalling/tiddit/main.nf [hunk 3]
```diff
@@ -1,35 +1,32 @@
...
+    tiddit_vcf_gz_tbi = TABIX_BGZIP_TIDDIT_SV.out.gz_tbi
+    tiddit_ploidy     = TIDDIT_SV.out.ploidy
+
+    ch_versions = ch_versions.mix(TABIX_BGZIP_TIDDIT_SV.out.versions)
+    ch_versions = ch_versions.mix(TIDDIT_SV.out.versions)
+
     emit:
     versions = ch_versions
+
+    tiddit_vcf = tiddit_vcf_gz_tbi
```
[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply subworkflows/nf-core/variantcalling/tiddit/main.nf:30]: What about this one?

File: workflows/sarek.nf (hunk 2 above)
[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply workflows/sarek.nf:762]: 👍

File: subworkflows/nf-core/variantcalling/tiddit/main.nf (hunk 3 above)
[the remark this answers is not in the document]
Comment (maxulysse) [reply subworkflows/nf-core/variantcalling/tiddit/main.nf:30]: oh yeah, I forgot about it, so we want only vcf.gz here, not the tbi?

[the remark this answers is not in the document]
Comment (maxulysse) [reply subworkflows/nf-core/variantcalling/tiddit/main.nf:30]: no worries

**Pull request merged**